lib/archive/tar/minitar/reader.rb

```diff
@@ -5,6 +5,7 @@ module Archive::Tar::Minitar
   # stream may be sequential or random access, but certain features only work
   # with random access data streams.
   class Reader
+    include Enumerable
```

Comment:
Enumerable provides `each_with_index`, `map`, `all?` and a bunch of convenient methods by reusing `each`.
http://ruby-doc.org/core-2.2.3/Enumerable.html